common/src/main/java/com/netflix/conductor/common/metadata/tasks/Task.java

```diff
@@ -198,6 +198,9 @@ public boolean isRetriable() {
     @ProtoField(id = 40)
     private int iteration;
 
+    @ProtoField(id = 41)
```

Comment:
We already have iteration as a part of the Task. Please evaluate if you can reuse that property.

Reply:
Thanks Manan. I checked that "iteration" was introduced as part of do-while task. So here is the problem in re-using the property. Do-while task and "Goto" interpret differently. The iteration is limited to the iteration of tasks running inside do-while container, while GOTO task consider the global state of this iteration. I am not sure whether re-using it will solve the problem. Let me know if there is any gap in my understanding.